synapse/rest/client/v2_alpha/shared_rooms.py

```diff
@@ -43,10 +43,10 @@ async def on_GET(self, request, user_id):
             raise SynapseError(
                 code=400,
                 msg="'user_id' must not be the authenticated user",
```

Comment:
```suggestion
                msg="'user_id' must not be the authenticated user",
```

I still think this wording is a bit weird. Perhaps something friendlier like "You cannot get shared rooms with yourself"?